src/Strategy/Python/Pipenv.hs

```diff
@@ -121,8 +132,9 @@ getDepsStatically ::
   PipenvProject ->
   m DependencyResults
 getDepsStatically project = context "Pipenv" $ do
-  lock <- context "Getting direct dependencies" $ readContentsJson (pipenvLockfile project)
-  graph <- context "Building dependency graph" $ pure (buildGraph lock Nothing)
+  lock <- context "Getting dependencies from Pipfile.lock" $ readContentsJson (pipenvLockfile project)
+  pipfile <- context "Getting dependencies from Pipfile" $ readContentsToml (pipenvPipfile project)
+  graph <- context "Building dependency graph" $ pure (buildGraph pipfile lock Nothing)
```

Comment:
Same comment re: using a `let` binding. I suppose the you could make the argument that the `context` is useful output for the customer, but I kind of don't think so. I'll leave both cases up to you.